src/map.js

```diff
@@ -0,0 +1,233 @@
+import React from "react";
+import L from "leaflet";
+import borderData from "./border.js";
+import leafletPip from "@mapbox/leaflet-pip";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      markerPosition: { lat: 44.0286, lng: -72.7317 },
+      oldMarkerPosition: { oldLat: 44.0886, oldLng: -72.7317 },
+      minimumZoom: 18,
+      maximumZoom: 18,
+      defaultZoom: 18,
+      gameStarted: false,
+      borderData: L.geoJSON(borderData),
+      currentPoint: { lat: 44.0286, lng: -72.7317 },
+      randomCord: { lat: 44.0286, lng: -72.7317 }
+    };
+
+    // binding the functions to the state
```

Comment:
You don't need to rebind the `this` if you define your methods as arrow functions (e.g. `startGame = () => { . . . }`) since arrow functions automatically bind the `this` to the containing scope.